src/lambda/commands/createNewSamApp.ts

```diff
@@ -66,6 +68,10 @@ export async function createNewSamApp(context: Pick<vscode.ExtensionContext, 'gl
     } else {
         await vscode.window.showTextDocument(uri)
     }
+
+    return new Map([
```

Comment:
Is it better to return a Map or an interface defined object here, for example
```
{
  runtime: config.runtime
}
```

Reply:
Looking at the rest of the PR, I assume the decision to use a Map was for convenience in telemetry. `createNewSamApp` should return something that makes the most sense for itself. I have a preference for a more first class type being returned, which could be transformed by the telemetry code as necessary.

Reply:
Adjusted to return an interface defined object